### PR TITLE
[USETUP] Localize the rest of the hardcoded error message strings

### PR DESCRIPTION
**Pull request by Fraizeraust**
## Purpose
Localize the respective string which has been added with the [`https://github.com/reactos/reactos/commit/35f7267a9712da2b94dfd2fc0c08d4a021f15b88`](https://github.com/reactos/reactos/commit/35f7267a9712da2b94dfd2fc0c08d4a021f15b88) commit. Meanwhile, since we're at it, localize the rest of the hard-coded strings as well.

### CODE REVIEW

File: base/setup/usetup/usetup.c [hunk 1]
```diff
@@ -3367,7 +3367,7 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
         if (!NT_SUCCESS(Status))
         {
             DPRINT1("BuildInstallPaths() failed. Status code: 0x%lx", Status);
-            PopupError("Failed to build the installation paths for the ReactOS installation directory!",
+            PopupError(MUIGetString(ERROR_NO_BUILD_PATH),
                        MUIGetString(STRING_CONTINUE),
```
Comment (MailYouLater) [inline base/setup/usetup/usetup.c]: In this commit, you add "ENTER = Reboot computer" to this message (now stored in `ERROR_NO_BUILD_PATH`) , so `STRING_CONTINUE` ("ENTER = Continue") should be removed.

File: base/setup/usetup/usetup.c [hunk 2]
```diff
@@ -3477,7 +3477,7 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
             if (!NT_SUCCESS(Status))
             {
                 DPRINT1("BuildInstallPaths() failed. Status code: 0x%lx", Status);
-                PopupError("Failed to build the installation paths for the ReactOS installation directory!",
+                PopupError(MUIGetString(ERROR_NO_BUILD_PATH),
                            MUIGetString(STRING_CONTINUE),
```
Comment (MailYouLater) [inline base/setup/usetup/usetup.c]: Ditto.

File: base/setup/usetup/lang/he-IL.h [hunk 3]
```diff
@@ -1478,6 +1478,11 @@ MUI_ERROR heILErrorEntries[] =
         "  \x07  .\204\220\227\232\204\204\216 \232\200\226\214 \211\203\213 F3 \225\207\214",
         "\204\220\227\232\204 \214\205\210\211\201 = F3  \212\231\216\204 = ENTER"
     },
+    {
+        // ERROR_NO_BUILD_PATH
+        "Failed to build the installation paths for the ReactOS installation directory!\n"
+        "ENTER = Reboot computer"
```
Comment (peterooch) [inline base/setup/usetup/lang/he-IL.h]: If you wouldn't mind, can you change to this:
`!ReactOS \214\231 \204\220\227\232\204\204 \232\211\227\211\232 \230\205\201\222 \204\211\227\211\232 \211\201\211\232\220 \232\211\211\220\201\201 \204\214\231\213\220 \204\220\227\232\204\204 \232\211\220\213\232\n`
`\201\231\207\216\204 \232\200 \231\203\207\216 \214\222\224\204 = ENTER`
Thanks.

Comment (Fraizeraust) [reply base/setup/usetup/lang/he-IL.h]: Sorry, that goes outside the purpose of what is this PR meant for. Could you please at least provide a screenshot confirming that the encoding is done properly?

Comment (peterooch) [reply base/setup/usetup/lang/he-IL.h]: How I can trigger this error?

File: base/setup/usetup/usetup.c [hunk 4]
```diff
@@ -1777,8 +1777,8 @@ SelectPartitionPage(PINPUT_RECORD Ir)
                 // &USetupData.SourceRootPath
                 if (RtlPrefixUnicodeString(&CurrentPartitionU, &USetupData.SourcePath, TRUE))
                 {
-                    PopupError("You cannot delete the partition containing the installation source!",
-                               MUIGetString(STRING_CONTINUE),
+                    PopupError(MUIGetString(ERROR_SOURCE_PATH),
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]:
```c
MUIDisplayError(ERROR_SOURCE_PATH, Ir, POPUP_WAIT_ENTER);
```

File: base/setup/usetup/usetup.c [hunk 5]
```diff
@@ -3569,8 +3569,8 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
         if (!NT_SUCCESS(Status))
         {
             DPRINT1("BuildInstallPaths() failed. Status code: 0x%lx", Status);
-            PopupError("Failed to build the installation paths for the ReactOS installation directory!",
-                       MUIGetString(STRING_CONTINUE),
+            PopupError(MUIGetString(ERROR_NO_BUILD_PATH),
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]:
```c
MUIDisplayError(ERROR_NO_BUILD_PATH, Ir, POPUP_WAIT_ENTER);
```

File: base/setup/usetup/usetup.c [hunk 6]
```diff
@@ -3582,8 +3582,8 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
          */
         if (RtlPrefixUnicodeString(&USetupData.SourcePath, &USetupData.DestinationPath, TRUE))
         {
-            PopupError("You cannot install ReactOS within the installation source directory!",
-                       MUIGetString(STRING_CONTINUE),
+            PopupError(MUIGetString(ERROR_SOURCE_DIR),
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]:
```c
MUIDisplayError(ERROR_SOURCE_DIR, Ir, POPUP_WAIT_ENTER);
```

File: base/setup/usetup/usetup.c [hunk 7]
```diff
@@ -3677,8 +3677,8 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
             if (!NT_SUCCESS(Status))
             {
                 DPRINT1("BuildInstallPaths() failed. Status code: 0x%lx", Status);
-                PopupError("Failed to build the installation paths for the ReactOS installation directory!",
-                           MUIGetString(STRING_CONTINUE),
+                PopupError(MUIGetString(ERROR_NO_BUILD_PATH),
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]:
```c
MUIDisplayError(ERROR_NO_BUILD_PATH, Ir, POPUP_WAIT_ENTER);
```

File: base/setup/usetup/usetup.c [hunk 8]
```diff
@@ -3690,8 +3690,8 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
              */
             if (RtlPrefixUnicodeString(&USetupData.SourcePath, &USetupData.DestinationPath, TRUE))
             {
-                PopupError("You cannot install ReactOS within the installation source directory!",
-                           MUIGetString(STRING_CONTINUE),
+                PopupError(MUIGetString(ERROR_SOURCE_DIR),
```
Comment (HBelusca) [inline base/setup/usetup/usetup.c]:
```c
MUIDisplayError(ERROR_SOURCE_DIR, Ir, POPUP_WAIT_ENTER);
```

**Pull request merged**